src/Stratis.Bitcoin.Features.SignalR/Broadcasters/BroadcasterBase.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Stratis.Bitcoin.AsyncWork;
+using Stratis.Bitcoin.Signals;
+using Stratis.Bitcoin.Utilities;
+
+namespace Stratis.Bitcoin.Features.SignalR.Broadcasters
+{
+    public abstract class ClientBroadcasterBase : IClientEventBroadcaster
+    {
+        private readonly EventsHub eventsHub;
+        private readonly INodeLifetime nodeLifetime;
+        private readonly ILoggerFactory loggerFactory;
+        private readonly AsyncProvider asyncProvider;
+        private readonly int broadcastFrequencySeconds;
+        protected readonly ILogger logger;
+        private IAsyncLoop asyncLoop;
+
+
+        protected ClientBroadcasterBase(
+            EventsHub eventsHub,
+            ISignals signals,
+            INodeLifetime nodeLifetime,
+            ILoggerFactory loggerFactory,
+            int broadcastFrequencySeconds = 5)
```

Comment:
Could we maybe make this part of the `signalRSettings`?

Reply:
Will do

Reply:
Done, well sort of, you can now specify the broadcastFrequency from the node e.g

``` options.ClientEventBroadcasters = new[]
                        {
                            (Broadcaster: typeof(StakingBroadcaster), ClientEventBroadcasterSettings: new ClientEventBroadcasterSettings
                                {
                                    BroadcastFrequencySeconds = 5
                                }),
                            (Broadcaster: typeof(WalletInfoBroadcaster), ClientEventBroadcasterSettings: new ClientEventBroadcasterSettings
                                {
                                    BroadcastFrequencySeconds = 5
                                })
                        };```
But @fassadlr thinks that @fenix2222 wanted this configurable from the command line?

Reply:
Leave it like that for now, we can do it via args later